model/task_queue.go

```diff
@@ -118,6 +118,7 @@ type TaskQueueItem struct {
 	ExpectedDuration    time.Duration `bson:"exp_dur" json:"exp_dur"`
 	Priority            int64         `bson:"priority" json:"priority"`
 	Dependencies        []string      `bson:"dependencies" json:"dependencies"`
+	CreateTime          time.Time     `bson:"create_time" json:"create_time"`
```

Comment:
Maybe I'm blind and don't see where it's used, but what is this key for?

Reply:
ah this is leftover from a different idea for how to implement